.github/workflows/unit-tests.yml

```diff
@@ -87,38 +87,3 @@ jobs:
           git commit -am "Look at this (photo)Graph *in the voice of Nickelback*" || true
           git push
           git push origin ${{ github.head_ref }}
-
-  npm-test-unit-components:
```

Comment:
This is deleted because the `billing.test.tsx` is covered in vitest which is the `npm run test:unit` workflow. Its all one runner for multiple file types.